src/main/java/br/com/tecnodev/entities/subCategory/NewSubCategoryForm.java

```diff
@@ -0,0 +1,85 @@
+package br.com.tecnodev.entities.subCategory;
+
+import br.com.tecnodev.entities.category.Category;
+
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Pattern;
+
+public class NewSubCategoryForm {
+
+    @NotEmpty(message = "{form.error.name}")
```

Comment:
Em validação de `String`, é  interessante utilizar o `@NotBlank`, pois além de checar se não é nulo nem vazio (como o `@NotEmpty)`, ele ainda faz um `trim()` para retirar os espaços vazios antes e depois. Você pode ler mais sobre isso [aqui](https://www.baeldung.com/java-bean-validation-not-null-empty-blank/).